install-raspbian.sh

```diff
@@ -114,6 +115,29 @@ else
     tar -xzvf /tmp/photobooth-latest.tar.gz -C /var/www/html/
 fi
 
+if [ -f "/etc/nginx/sites-enabled/default" ]; then
+    info "### Enable PHP in NGINX"
+    cp /etc/nginx/sites-enabled/default ~/nginx-default.bak
+    sed -i 's/^\(\s*\)index index\.html\(.*\)/\1index index\.php index\.html\2/g' /etc/nginx/sites-available/default
+    sed -i '/location ~ \\.php$ {/s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
+    sed -i '/include snippets\/fastcgi-php.conf/s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
+    sed -i '/fastcgi_pass unix:\/run\/php\//s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
+    sed -i '/.*fastcgi_pass unix:\/run\/php\//,// { /}/s/^\(\s*\)#/\1/g; }' /etc/nginx/sites-available/default
+
+    info "### Testing NGINX config"
+    /usr/sbin/nginx -t -c /etc/nginx/nginx.conf &>/dev/null && RESULT='config test ok' || RESULT='config test failed'
+
+    if [[ $RESULT == 'config test ok' ]]; then
+        info "$RESULT"
+        info "### Restarting NGINX"
+        /etc/init.d/nginx restart
+    else
+        error "$RESULT"
+    fi
```

Comment:
No need for this check. If the rest run fails the script will abort anyway, due to `set -e`.